exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/WebServer.java

```diff
@@ -96,6 +107,23 @@
 
   private Server embeddedJetty;
 
+  private static File tmpJavaScriptDir;
+
+  public File getTmpJavaScriptDir() {
+    if (tmpJavaScriptDir == null) {
+      tmpJavaScriptDir = Files.createTempDir();
+      tmpJavaScriptDir.deleteOnExit();
+      //Perform All auto generated files at this point
+      try {
+        generateOptionsDescriptionJSFile();
+      } catch (IOException e) {
+        // TODO Auto-generated catch block
+        e.printStackTrace();
```

Comment:
Please remove this and throw appropriate exception.

Reply:
Ok.